src/DependencyInjection/Factory/RoutingAdminFactory.php

```diff
@@ -35,16 +35,22 @@ public function getKey()
     public function addConfiguration(NodeBuilder $builder)
     {
         $builder->scalarNode('basepath')->defaultNull()->end();
+        $builder->scalarNode('content_basepath')->defaultNull()->end();
     }
 
     /**
      * {@inheritdoc}
      */
     public function create(array $config, ContainerBuilder $container, XmlFileLoader $loader)
     {
+        if (!$config['enabled']) {
```

Comment:
ah ok. fixed it. could we also check for enabled before calling process? that bit looks a bit too implicit for my taste (we check if one of the parameters that would be set if enabled is indeed set)